Providers/Modules/Plugins/PatchManagement/conf/patch_management_inventory.mof

```diff
@@ -2,14 +2,6 @@
 @TargetNode='Localhost'
 */
 
-instance of MSFT_nxPackageResource
-{
-    Name = "*";
-    ResourceId = "[MSFT_nxPackageResource]Inventory";
-    ModuleName = "PSDesiredStateConfiguration";
-    ModuleVersion = "1.0";
-};
```

Comment:
For limited time, for the Patch Management solution, we have decided to stop sending the _Installed Packages_ info and thus, I updated the inventory MOF to not have the **MSFT_nxPackageResource** entry. It is still used by the change tracking solution.